tests/cancel-test.js

```diff
@@ -1,17 +1,17 @@
-import Backburner from "backburner";
+import Backburner from 'backburner';
```

Comment:
@stefanpenner should these single quotes be included in this PR? Could they be in a subsequent PR?

They're noisy and making it hard to tell where your main changes are.